Xenon/Core/XCPU/IIC/IIC.cpp

```diff
@@ -204,7 +208,9 @@ void Xe::XCPU::IIC::XenonIIC::genInterrupt(u8 interruptType,
   for (u8 ppuID = 0; ppuID < 6; ppuID++) {
     if ((cpusToInterrupt & 0x1) == 1) {
 
-      LOG_DEBUG(Xenon_IIC, "Generating interrupt: Thread {}, intType: {}", ppuID, getIntName(interruptType));
+      if (IIC_DEBUG) {
```

Comment:
Make this use ifdefs, not a #if debug, true false system
#ifdef IIC_DEBUG
//
#endif // IIC_DEBUG